Flutter/en/1.4.3.md

```diff
@@ -0,0 +1,5 @@
+# Success Criterion 1.4.3 - Contrast (Minimum)
+
+When designing an app, take the contrast ratio of all text into account. The contrast ratio of text should be at least 4,5:1, or 3:1 for large text.
+
+You can take a screenshot of the app to determine colors and calculate contrast.
```

Comment:
Also add info about packages / IDE plugins if there are any added in 1.4.11

Reply:
Added the advice to use a online contrast calculator

Reply:
Let me know if you think this needs some more information.